### PR TITLE
Add deployment to the quick start 

### PR DESCRIPTION
**Pull request by venusang**
This PR addresses  issue: Add deployment to the quick start https://github.com/ember-learn/guides-source/issues/1320.

The changes involved include:

- Step by step instructions on how to deploy to Netlify in 2 ways (drag and drop, GitHub)
- Corresponding screenshots related to each step

### CODE REVIEW

File: guides/release/getting-started/quick-start.md [hunk 1]
```diff
@@ -366,4 +367,76 @@ add the following directive to the application's virtual host configuration:
...
-<!-- eof - needed for pages that end in a code block  -->
+## Deploying your app to Netlify
+
+[Netlify](http://netlify.com/products) is a platform that deploys your apps to production.
+
+![About Netlify](/images/quick-guide/netlify/netlify-product.png)
+
+Why Netlify?
+
+It's fun and easy! It does not require a high level of knowledge for you to deploy your website to production. Netlify offers a free account option that satisfies most website production needs and no credit card is required. Netlify is also a long-term supporter of Ember ❤️. Following these steps will hopefully help you get your site up and running in minutes.
```
Comment (jenweber) [inline guides/release/getting-started/quick-start.md]:
```suggestion
It does not require a high level of knowledge for you to deploy your website to production.
Netlify offers a free account option and no credit card is required.
These Guides themselves are hosted on Netlify, while other parts of `emberjs.com` are hosted using Heroku, Fastly, GitHub pages, and AWS.
Overall, Ember developers have many options for how they deploy their apps!

Following these steps will help you get your site up and running in minutes:
```

File: guides/release/getting-started/quick-start.md [hunk 2]
```diff
@@ -366,4 +367,76 @@ add the following directive to the application's virtual host configuration:
...
+
+First you need to [sign-up for a Netlify account](https://app.netlify.com/signup) account if you do not already have one:
```
Comment (jenweber) [inline guides/release/getting-started/quick-start.md]:
```suggestion
First you need to [sign up for a Netlify account](https://app.netlify.com/signup) account if you do not already have one:
```

File: guides/release/getting-started/quick-start.md [hunk 3]
```diff
@@ -366,4 +367,81 @@ add the following directive to the application's virtual host configuration:
...
+Why Netlify?
+
+It does not require a high level of knowledge for you to deploy your website to production.
+Netlify offers a free account option and no credit card is required.
+These Guides themselves are hosted on Netlify, while other parts of `emberjs.com` are hosted using Heroku, Fastly, GitHub pages, and AWS.
+Overall, Ember developers have many options for how they deploy their apps!
+
+Following these steps will help you get your site up and running in minutes:
+
+First you need to [sign up for a Netlify account](https://app.netlify.com/signup) account if you do not already have one:
```
Comment (jenweber) [inline guides/release/getting-started/quick-start.md]:
```suggestion
First you need to [sign up for a Netlify account](https://app.netlify.com/signup) if you do not already have one:
```

**Comment by jenweber** [comment]
There is some discussion of whether we should copy this into the deployment section of the guides and then link to it, instead of putting it in the quickstart directly. Maintainers, please hold off on merging until that is decided.

File: guides/release/getting-started/quick-start.md [hunk 4]
```diff
@@ -366,4 +367,81 @@ add the following directive to the application's virtual host configuration:
...
+[Netlify](http://netlify.com/products) is a one of many ways to deploy your app to the web so you can share it with others!
+
+![About Netlify](/images/quick-guide/netlify/netlify-product.png)
+
+Why Netlify?
+
+It does not require a high level of knowledge for you to deploy your website to production.
+Netlify offers a free account option and no credit card is required.
+These Guides themselves are hosted on Netlify, while other parts of `emberjs.com` are hosted using Heroku, Fastly, GitHub pages, and AWS.
+Overall, Ember developers have many options for how they deploy their apps!
```
Comment (jenweber) [inline guides/release/getting-started/quick-start.md]:
```suggestion
Overall, Ember developers have many options for how they deploy their apps! You can learn more about them in the [Deployment tutorial](../tutorial/deploying/#toc_deployment-options).
```

File: guides/release/getting-started/quick-start.md [hunk 5]
```diff
@@ -383,6 +384,84 @@ add the following directive to the application's virtual host configuration:
...
+[Netlify](http://netlify.com/products) is a one of many ways to deploy your app to the web so you can share it with others!
+
+![About Netlify](/images/quick-guide/netlify/netlify-product.png)
+
+Why Netlify?
+
+It does not require a high level of knowledge for you to deploy your website to production.
+Netlify offers a free account option and no credit card is required.
+These Guides themselves are hosted on Netlify, while other parts of `emberjs.com` are hosted using Heroku, Fastly, GitHub pages, and AWS.
+Overall, Ember developers have many options for how they deploy their apps! You can learn more about them in the [Deployment tutorial](../tutorial/deploying/#toc_deployment-options).
```
Comment (MelSumner) [inline guides/release/getting-started/quick-start.md:398]:
```suggestion
Overall, Ember developers have many options for how they deploy their apps! Netlify is just one of the many options you have.
```

**Pull request merged**